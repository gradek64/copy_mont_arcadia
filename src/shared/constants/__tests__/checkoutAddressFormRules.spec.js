import rules from '../checkoutAddressFormRules'

describe('Checkout Address Form Rules', () => {
  it('Provie address form rules', () => {
    expect(rules).toMatchSnapshot()
  })
  describe('UK postcode', () => {
    const { pattern } = rules['United Kingdom']
    describe('accepts valid patterns', () => {
      it('EC1 1AA', () => {
        expect('EC1 1AA'.match(pattern)).toBeTruthy()
      })
      it("'EC1 1AA  '", () => {
        expect('EC1 1AA  '.match(pattern)).toBeTruthy()
      })
      it('EC11AA', () => {
        expect('EC11AA'.match(pattern)).toBeTruthy()
      })
    })
    it("doesn't match invalid postcode", () => {
      expect('x'.match(pattern)).toBeFalsy()
    })
  })
})
