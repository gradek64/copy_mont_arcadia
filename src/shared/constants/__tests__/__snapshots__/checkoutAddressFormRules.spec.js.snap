// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout Address Form Rules Provie address form rules 1`] = `
Object {
  "Albania": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Andorra": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Anguilla": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Argentina": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Armenia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Australia": Object {
    "pattern": "^ *[0-9]{4} *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": "input",
  },
  "Austria": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Azerbaijan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Bangladesh": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Barbados": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Belgium": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Bermuda": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Bolivia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Bosnia and Herzegovina": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Brazil": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Brunei": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Bulgaria": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Cambodia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Canada": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": "input",
  },
  "Cayman Islands": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Chile": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "China": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Colombia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Croatia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Czech Republic": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Denmark": Object {
    "pattern": "^ *[^\\\\s]*[0-9]{3,4} *$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Estonia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Falkland Islands": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Faroe Islands": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Finland": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "France": Object {
    "pattern": "^ *[0-9]{5} *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "French Guiana": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Germany": Object {
    "pattern": "^ *[0-9]{5} *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Greece": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Greenland": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Guadeloupe": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Guernsey": Object {
    "pattern": "^ *[Gg][Yy]([0-9]|10)[ ]?[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2} *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "House name or number",
    "premisesRequired": false,
    "stateFieldType": false,
  },
  "Holy See": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Hong Kong S.A.R. of China": Object {
    "pattern": ".",
    "postcodeLabel": "Town or Locality",
    "postcodeRequired": false,
    "premisesLabel": "Street Address",
    "premisesRequired": false,
    "stateFieldType": false,
  },
  "Hungary": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "India": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Index Number",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Indonesia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Ireland": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Town or Locality",
    "postcodeRequired": false,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Italy": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Japan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Jersey": Object {
    "pattern": "^ *[Jj][Ee]{1,2}[0-5][ ]?[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2} *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "House name or number",
    "premisesRequired": false,
    "stateFieldType": false,
  },
  "Kazakhstan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Korea South": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Kyrgyzstan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Latvia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Libya": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Liechtenstein": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Lithuania": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Luxembourg": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Madagascar": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Malaysia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Maldives": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Marshall Islands": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Mexico": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Moldova": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Monaco": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Montenegro": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Morocco": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Nepal": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Netherlands": Object {
    "pattern": "^ *[0-9]{4}[\\\\s][a-zA-Z]{2} *$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "New Caledonia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "New Zealand": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Niger": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Norway": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Oman": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Pakistan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Palau": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Philippines": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Zip Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Poland": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Portugal": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Puerto Rico": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Romania": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "S Georgia & S Sandwich Islands": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Saint Pierre and Miquelon": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Samoa": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "San Marino": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Serbia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Singapore": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Slovakia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "South Africa": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Spain": Object {
    "pattern": "^ *([a-zA-Z]{2}|[0-9]{2})[0-9]{3} *$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": "input",
  },
  "Sri Lanka": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "St Vincent and the Grenadines": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Svalbard and Jan Mayen": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Sweden": Object {
    "pattern": "^ *[1-9][0-9]{2}[\\\\s][0-9]{2} *$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Switzerland": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Taiwan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Thailand": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Togo": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Tunisia": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Turkey": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Turkmenistan": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Ukraine": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "United Kingdom": Object {
    "pattern": "^ *(([gG][iI][rR] {0,}0[aA]{2})|(([aA][sS][cC][nN]|[sS][tT][hH][lL]|[tT][dD][cC][uU]|[bB][bB][nN][dD]|[bB][iI][qQ][qQ]|[fF][iI][qQ][qQ]|[pP][cC][rR][nN]|[sS][iI][qQ][qQ]|[iT][kK][cC][aA]) {0,}1[zZ]{2})|((([a-pr-uwyzA-PR-UWYZ][a-hk-yxA-HK-XY]?[0-9][0-9]?)|(([a-pr-uwyzA-PR-UWYZ][0-9][a-hjkstuwA-HJKSTUW])|([a-pr-uwyzA-PR-UWYZ][a-hk-yA-HK-Y][0-9][abehmnprv-yABEHMNPRV-Y]))) {0,}[0-9][abd-hjlnp-uw-zABD-HJLNP-UW-Z]{2})) *$",
    "postcodeLabel": "Postcode",
    "postcodeRequired": true,
    "premisesLabel": "House number",
    "premisesRequired": false,
    "stateFieldType": false,
  },
  "United States": Object {
    "pattern": "^ *[0-9]{5}(-[0-9]{4}){0,1} *$",
    "postcodeLabel": "Zip Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": "SELECT",
  },
  "Uruguay": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Vietnam": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
  "Virgin Islands British": Object {
    "pattern": "^ *[\\\\w ]+$",
    "postcodeLabel": "Postal Code",
    "postcodeRequired": true,
    "premisesLabel": "Street Address",
    "premisesRequired": true,
    "stateFieldType": false,
  },
}
`;
